Split/ExportedImports.swift

```diff
@@ -0,0 +1,3 @@
+#if SWIFT_PACKAGE
```

Comment:
ℹ️ This directive command is to prevent buidl failures when consuming through CocoaPods